python/ray/scripts/scripts.py

```diff
@@ -1774,21 +1774,40 @@ def install_nightly(verbose, dryrun):
 
 @cli.command()
 @click.option(
-    "--output", "-o", required=False, type=str, help="Output directory.")
-def get_cpp_example(output):
-    """Copy cpp example project to output directory."""
+    "--copy-project-to",
+    "-cp",
+    required=False,
+    type=str,
+    help="Output directory.")
+def get_cpp_library(copy_project_to):
+    """Get the paths of cpp library and example project."""
     raydir = os.path.abspath(os.path.dirname(ray.__file__))
-    if not output:
-        output = "."
-    if not os.path.isdir(output):
-        raise Exception(
-            "Please give an existing output directory by '--output' or '-o'.")
-    input = os.path.join(raydir, "core/src/ray/cpp/example")
-    output_pro = os.path.join(output, os.path.basename(input))
-    shutil.copytree(input, output_pro)
-    print(
-        f"C++ example project has been copied to {os.path.abspath(output_pro)}"
-    )
+    cpp_example_dir = os.path.join(raydir, "core/src/ray/cpp/example")
+    cpp_library_dir = os.path.join(cpp_example_dir, "thirdparty")
+    welcome_msg = "Welcome to use ray C++ API."
+    print("-" * len(welcome_msg))
+    print("Welcome to use ray C++ API.")
+    print("-" * len(welcome_msg))
+    print("\nYou can find ray C++ library in ")
+    print(f"    {cpp_library_dir}")
+    print("Please include and link this library in your projects by yourself.")
+    print("\nOr, you can use our template bazel project directly."
+          "You can find it in ")
+    print(f"    {cpp_example_dir}")
```

Comment:
```suggestion
    print("Please include the headers and link the library file in your project manually.")
    print("\nOr, you can create a new Ray C++ application starting from our bazel template project located in")
    print(f"    {cpp_example_dir}")
```

PS: We should inform users that only `api.h` is needed to be included in source files.